design/ErrorHandling.md

```diff
@@ -23,6 +23,148 @@ There are several types of WebGPU calls that get their errors handled differentl
 Implementations should provide a way to enable synchronous validation, for example via a debug shim or via the developer tools.
 The extra overhead needs to be low enough that applications can still run while being debugged.
 
+## *Fatal Errors*: Lost/Recovered Events
+
+<!-- calling this revision 5 -->
```

Comment:
does there need to be revision index in the text?

Reply:
I didn't actually save all of the revisions, so I can't really make one, unfortunately.